CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1]
+### Fixed
```

Comment:
New addition to main branch (https://github.com/MetaMask/create-release-branch/compare/v3.0.0...HEAD):

```suggestion
## [3.0.1]
### Changed
- Bump `@metamask/utils` to `^9.0.0` ([#150](https://github.com/MetaMask/create-release-branch/pull/150))

### Fixed
```